### ISSUE TITLE
Scrape external links posted in Reddit

### ISSUE THREAD

**Issue by syltruong** [ahmedbesbes/media-agent#6]
In Reddit, many submissions will relay news from other traditional media and include links to those.

These are the places where the most information lies. It would be great to be able to embed that information.

**Comment by ahmedbesbes** [comment]
I think the first thing to do would be to extract the answers of each Reddit submission since they contain a lot of relevant information to the post.

**Comment by syltruong** [comment]
Yes indeed, it is totally feasible. This is the [reference](https://praw.readthedocs.io/en/stable/tutorials/comments.html#extracting-comments) for navigating the comments of a post.

I can suggest to pick this task up this weekend

**Comment by ahmedbesbes** [comment]
It'd be great, thank you!
